apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts

```diff
@@ -148,4 +196,170 @@ export class ManageGroupsAccountsComponent implements OnInit, OnDestroy, AfterVi
 		this.breadcrumbUniquePath = obj.breadcrumbUniqueNamePath;
 		this.breadcrumbPath = obj.breadcrumbPath;
 	}
+
+    /**
+     * To submit custom field data
+     *
+     * @param {*} type API call operation type
+     * @param {*} value
+     * @memberof ManageGroupsAccountsComponent
+     */
+    public submitCustomFields(value: any): void {
+        this.isSaveCustomInProgress = true;
+        this.groupService.createCompanyCustomField(value.customField).subscribe(response => {
+            if (response) {
+                if (response.status === 'success') {
+                    this.customFieldForm.get('customField').reset();
+                    let customFieldResponse = response.body;
+                    this.updateModeLength = customFieldResponse.length;
+                    this.renderCustomField(customFieldResponse);
+                    this.toasterService.successToast('Custom field updated successfully');
+                } else {
+                    this.toasterService.errorToast(response.message);
+                }
+                this.isEnabledIndex = null;
+                this.isSaveCustomInProgress = false;
+                if (this.modalRef) {
+                    this.modalRef.hide()
+                }
+            }
+        });
+    }
+
+    /**
+     * API call to get custom field data
+     *
+     * @memberof ManageGroupsAccountsComponent
+     */
+    public getCompanyCustomField(): void {
+        this.isGetCustomInProgress = true;
+        this.groupService.getCompanyCustomField().subscribe(response => {
+            this.isEnabledIndex = null;
+            if (response && response.status === 'success') {
+                this.renderCustomField(response.body);
+                this.updateModeLength = response.body.length;
+            } else {
+                this.toasterService.errorToast(response.message);
```

Comment:
If response is null then this would break

Reply:
this block is already inside response nullity check